extension/test/unit/addProperties-test.js

```diff
@@ -0,0 +1,68 @@
+const assert = require('assert')
+const proxyquire = require('proxyquire')
+
+const getProductsResult = () => ({
+  products: [
+    {
+      id: '1',
+      properties: [
+        {
+          label: 'added',
+          value: 'added'
+        },
+        {
+          label: 'ignored',
+          value: 'ignored'
+        },
+        {
+          label: 'foo',
+          value: 'bar'
+        }
+      ]
+    }
+  ]
+})
+
+describe('addProperties', () => {
+  it('should add properties if they are configured', async () => {
+    let subjectUnderTest = proxyquire('./../../addProperties', {
+      './config': {
+        addProperties: 'foo,added'
+      }
+    })
+
+    const { products } = await subjectUnderTest({}, getProductsResult())
+    assert.deepEqual(products[0].additionalProperties, [
+      {
+        label: 'added',
+        value: 'added'
+      },
+      {
+        label: 'foo',
+        value: 'bar'
+      }
+    ])
+  })
+
+  it('should not do anything if no property is matched', async () => {
```

Comment:
this is case sensitive implementation, what do you think about case insensitive implementation? we could ask Daniela..

Reply:
makes sense, added